doc/vim-go.txt

```diff
@@ -1700,6 +1700,33 @@ example sometimes code.google.com times out. To test, just execute a simple
 You'll see a more detailed error. If this works, vim-go will work too.
 
 
+I want to use a different binary name than "go", can I do this?~
+
+There is no way to directly configure the binary name; but you can use a
+wrapper script; for example if you would like to run `goapp` instead of `go`:
+
+1. In `~/gobin/go` (remember to make it executable):
+>
+     #!/bin/sh
+     # Remove gobin from PATH and run goapp.
+     PATH=${PATH#$HOME/gobin} goapp "$@"
+<
+2. Start Vim with `~/gobin` as the first `PATH` entry so it will use the
+   wrapper script:
+>
+     PATH="$HOME/gobin/:$PATH" vim
+<
+   Or from within your vimrc:
+>
+     augroup go_path
+       autocmd!
+       autocmd FileType go 
+         \  if stridx($PATH, expand("$HOME/gobin")) == -1
+         \|   let $PATH = expand("$HOME/gobin:$PATH")
+         \| endif
+     augroup end
+<
+
```

Comment:
I'm ok for the FAQ entry, but we don't nee the `autocmd` tip :) The wrapper should be enough.